proto/greenfield/storage/tx.proto

```diff
@@ -342,10 +342,7 @@ message MsgGroupMember {
   // member defines the account address of the group member
   string member = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
   // expiration_time defines the expiration time of the group member
-  google.protobuf.Timestamp expiration_time = 2 [
-    (gogoproto.stdtime) = true,
-    (gogoproto.nullable) = false
-  ];
+  google.protobuf.Timestamp expiration_time = 2 [(gogoproto.stdtime) = true];
```

Comment:
why change this?

Reply:
So the expiration can be nil (means no expiration)

Reply:
do this change still needed? if Change this, off-chain service should adapt to this, `nil == expiration`

@Pythonberg1997 @guagualvcha

Reply:
or we can set nil to maxTimestamp when getting members from rpc, producing events